BetaCameras/RealSense2/RealSense2.cs

```diff
@@ -1681,38 +1607,18 @@ protected override void DeactivateChannelImpl(String channelName)
                     return;
                 }
 
-                stream = RealSense2API.Stream.DEPTH;
+                stream = Stream.Depth;
             }
             else if (channelName == ChannelNames.Left)
             {
-                stream = RealSense2API.Stream.INFRARED;
+                stream = Stream.Infrared;
```

Comment:
Deactivating Left also deactivates Right. Does `DisableStream` support an `index` parameter?
Otherwise create an issue and we decide if/how we will fix it.

Reply:
Yes, it does support an index. Fixed.